src/EzPlatformEncoreBundle/bundle/DependencyInjection/EzSystemsEzPlatformEncoreExtension.php

```diff
@@ -15,67 +15,36 @@
 
 class EzSystemsEzPlatformEncoreExtension extends Extension
 {
-    const EZ_ENCORE_CONFIG_NAME = 'ez.config.js';
-    const EZ_ENCORE_MANAGER_NAME = 'ez.config.manager.js';
+    const CONFIG_NAMES = [
+        'ez.config.js',
+        'ez.config.manager.js',
+        'ez.webpack.config.js',
```

Comment:
I created a PR for automatically including this into the webpack configs (https://github.com/ezsystems/ezplatform/pull/382). However, I used a different name for the file: `ez.webpack.custom.config.js`. Could you update your PR?

Reply:
Sure ;)